### ISSUE TITLE
[bug] Fix order

### ISSUE THREAD

**Issue by thomaLV** [eslint/eslint#11828]
**Tell us about your environment**

* **ESLint Version:** 5.16.0
* **Node Version:** 12.4.0
* **npm Version:** 6.9.0

**Please show your full configuration:**

<details>
<summary>Configuration (excerpt)</summary>

```js
"extends": "eslint:recommended",
"rules": {
	"template-curly-spacing": ["warn", "never"],
	"no-multi-spaces": "warn",
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
recurse(cur[i], `${prop  }[${  i  }]`);
```

Running in vscode I clicked "Fix all template-curly-spacing problems" as shown in the picture below.
![image](https://user-images.githubusercontent.com/36425287/59304409-81140000-8c98-11e9-8dca-4952c2486c0c.png)

**What did you expect to happen?**
```js
recurse(cur[i], `${prop}[${i}]`);
```

**What actually happened? Please include the actual, raw output from ESLint.**
```js
recurse(cur[i], `${prop }[${ i }]`);
```

**Comment by mysticatea** [comment]
Thank you for your report.

However, this is an issue on `vscode-eslint` rather than ESLint. Please open an issue on https://github.com/microsoft/vscode-eslint.

**Comment by thomaLV** [comment]
Thank you, I'll do that
